File: PHP/Car.php
<?php

require_once('Account.php');

class Car {
  private $id;
  private $license;
  private $driver;
  private $passenger;

  public function __construct($id, $license, Account $driver, $passenger) {
    $this->id = $id;
    $this->license = $license;
    $this->driver = $driver;
    $this->passenger = $passenger;
  }
}

File: python/car.py
from account import Account
class Car:
  id = int
  license = str
  driver = Account("","","","", "")
  passanger = int

  def __init__(self, id, license, driver, passanger):
    self.id = id
    self.license = license
    self.driver = driver
    self.passanger = passanger